openghg/processing/_process_footprint.py

```diff
@@ -274,25 +274,31 @@ def align_datasets(
 
     if platform is not None:
         platform = platform.lower()
-        if platform in ("satellite", "flask"):
+        # Do not apply resampling for "satellite" (but have re-included "flask" for now)
+        if platform in ("satellite"):
```

Comment:
This could just be a simple comparison

Reply:
It could, but was thinking we'd may need to expand this output to be set of values again in the future?

Reply:
OK, I changed it with 0e0c5c687f9689ee8c42c0ad1255a3ddb96eebd2 before seeing this, we can change it back when we need to.